node/engine/messageservice/messageservice.go

```diff
@@ -1,11 +1,16 @@
 // Package messageservice is a messaging service responsible for routing messages to peers and relaying messages received from peers.
 package messageservice // import "github.com/statechannels/go-nitro/node/messageservice"
 
-import "github.com/statechannels/go-nitro/protocols"
+import (
+	p2pms "github.com/statechannels/go-nitro/node/engine/messageservice/p2p-message-service"
+	"github.com/statechannels/go-nitro/protocols"
+)
 
 type MessageService interface {
-	// Out returns a chan for receiving messages from the message service
-	Out() <-chan protocols.Message
+	// P2PMessages returns a chan for receiving messages from the message service
+	P2PMessages() <-chan protocols.Message
+	// SignRequests returns a chan for receiving signature requests from the message service
+	SignRequests() <-chan p2pms.SignatureRequest
```

Comment:
Do we want the general `messageservice` interface to depend on a type from the `p2pms` package? Maybe we should hoist this type up to this package, instead? That would also prevent `test-messageservice` depending on `p2pms`.